### PR TITLE
MangaDex to MangaDex backup migrator

### PR DESCRIPTION
**Pull request by FramboisePi**
Add a migrator replacing MangaDex legacy ids by their corresponding UUIDs.

Capture d’écran 2021-05-23 à 09 07 49

### CODE REVIEW

File: src/tools/md-to-md-migrator.md [hunk 1]
```diff
@@ -0,0 +1,30 @@
+---
+title: MangaDex backup migrator
+lang: en-US
+---
+
+:::: guide MangaDex to MangaDex backup migrator
+MangaDex changed their ids to UUIDS. If the source remains compatible with legacy ids, using them will be slower as it requires to call a conversion endpoint every time.
+
+To migrate you library:
```
Comment (AdrienSeon) [inline src/tools/md-to-md-migrator.md:9]: Your*

File: src/tools/md-to-md-migrator.md [hunk 2]
```diff
@@ -0,0 +1,30 @@
...
+1. [Make a backup](/help/faq/#how-can-i-make-a-backup-of-my-library) of your library
+1. Convert your backup with this *MangaDex to MangaDex backup migrator*
+1. Reinstall Paperback
+1. [Restore](/help/faq/#how-can-i-make-a-backup-of-my-library) the migrated backup
+
+::: aside
+Please note the conversion can take some time on big libraries
```
Comment (AdrienSeon) [inline src/tools/md-to-md-migrator.md:16]: Add "that" after note maybe ?

**Review by AdrienSeon — COMMENTED** [review]
Here is 2 little comments regarding the wording
Other than that, i'm not familiar enough with vuepress to review the code
Good work !

File: src/tools/md-to-md-migrator.md (hunk 2 above)
Comment (FramboisePi) [reply src/tools/md-to-md-migrator.md]: Fixed by 11223d8

File: src/tools/md-to-md-migrator.md (hunk 1 above)
Comment (FramboisePi) [reply src/tools/md-to-md-migrator.md]: Thank you for your review Ankah! Fixed by 11223d8

**Pull request merged**